solr/solr-ref-guide/src/collection-management.adoc

```diff
@@ -264,16 +381,33 @@ processing of updates and reloads the collection.
 
 Fetch the names of the collections in the cluster.
 
-`/admin/collections?action=LIST`
-
-=== Examples using LIST
+[.dynamic-tabs]
+--
+[example.tab-pane#v1listcollection]
+====
+[.tab-label]*V1 API*
 
-*Input*
+With the v1 API, the `list` command must be capitalized as `LIST`:
 
-[source,text]
+[source,bash]
 ----
 http://localhost:8983/solr/admin/collections?action=LIST
 ----
+====
+
+[example.tab-pane#v2listcollection]
+====
+[.tab-label]*V2 API*
+
+With the v2 API, the `modify` command is provided as part of the JSON data that contains the required parameters:
```

Comment:
> the `modify` command

  Copy/paste error maybe?

Reply:
Can you elaborate?  I checked and V1 is "modifycollection", but in v2 we shortened it to "modify", at least accoridng to the introspection of the api and my quick test!

Reply:
Oh, yep, sorry.

The actual example API call below is hitting the list-collections API, but the description text is talking about the "modify" command.

Put differently, should this read "the `list` command" instead of "the `modify` command"

Reply:
Doh!   right, we don't need this sentence all, cause it is one of the few v2 commands that actually behaves the way a RESTful command should ;-)